projects/noloan/admin/src/main/java/hasadna/noloan/admin/app/SuggetionRecyclerAdapter.java

```diff
@@ -141,7 +130,6 @@ public void bind(SmsMessage sms) {
             FirestoreClient client = new FirestoreClient();
 
             SmsMessage aprroved = sms.toBuilder().setApproved(true).build();
-            Log.d("!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!", aprroved.getApproved() + "");
             client.modifyMessage(sms, aprroved);
 
             //client.writeMessage(sms);
```

Comment:
Test comment?

BTW - So in the admin app... We don't use `DbMessages` as our "tool" to pass changes to the database as we do in the NoLoan app?
Meaning... We pass the changes directly via the FirestoreClient? Why is it different?

Reply:
Yes. also in the app.
We don't use DbMessages to pass to the DB. It just hold the messages from there.
FirestoreClient is the tool to pass changes to the DB

Reply:
>Yes. also in the app.
We don't use DbMessages to pass to the DB. It just hold the messages from there.
FirestoreClient is the tool to pass changes to the DB

The way the app goes (as it is in the master branch)

All changes concerning the DB are done via the `SmsMessages`, which passes the changes OR not (according to the logic inside the `SmsMessages` functions) to the FirestoreClient.

Please mention that this have been changed in you PR.

Reply:
Hey Gal, the purpose of this PR is to share common code between the main and admin app, but it seems you're also trying to do more stuff here like change the way `DbMessages` work.
If that's the case, please focus this PR just on the narrow goal of sharing code, and any other changes - put in another PR (or several PRs, depending on the change).